src/components/apps/launch.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @author psarando
+ *
+ * A component for displaying the App Launch Wizard for a given app.
+ * Also handles calling analysis submission and quick-launch-save endpoints.
```

Comment:
I moved some common logic that was in the app launch page and also in the new relaunch page into this component.

Depending on what we decide in our UX review meeting, this file could be renamed `src/components/apps/launch/index.js`.